floyd/cli/data_upload_utils.py

```diff
@@ -49,7 +49,7 @@ def upload_is_resumable(data_config):
     )
 
 
-def initialize_new_upload(data_config, access_token, description=None):
+def initialize_new_upload(data_config, access_token, source_dir='.', description=None):
```

Comment:
better add new keyword arguments to the end so it won't break users who are using existing keyword args as positional arguments.

Reply:
Agreed, updated this in the latest commit.